src/core/fonts.js

```diff
@@ -1258,12 +1301,14 @@ var Font = (function FontClosure() {
       return shadow(this, "renderer", renderer);
     },
 
-    exportData: function Font_exportData() {
-      // TODO remove enumerating of the properties, e.g. hardcode exact names.
-      var data = {};
-      for (var i in this) {
-        if (this.hasOwnProperty(i)) {
-          data[i] = this[i];
+    exportData() {
+      const data = Object.create(null);
+      let property, value;
```

Comment:
I think we can remove this line if we use `for (const property ...` and `const value = this...` below.

Reply:
Actually, that seems *slightly* slower based on bot run-time hence I'm reverting this change (besides those variables will get reused with the follow-up PR anyway).